src/main/resources/application.yaml

```diff
@@ -58,6 +58,31 @@ stargate:
           url: https://us-central1-aiplatform.googleapis.com
           api-key: YOUR_API_KEY
 
+        google:
+          enabled: true
+          url: "https://us-central1-aiplatform.googleapis.com/v1/projects/{PROJECT_ID}/locations/us-central1/publishers/google/models/{MODEL}:predict"
+          supported-authentication:
+            - "HEADER"
+            - "SHARED_SECRETS"
+          parameters:
+            PROJECT_ID:
```

Comment:
We may need to create provider conjugates, can't we have PROJECT as a field value like name: "PROJECT_ID"

Reply:
I think there will be a problem if we have multiple parameters:
```
  parameters:
    name: PROJECT_ID
    type: STRING
    required: true
    name: Another_parameter
    type: STRING
    required: true
```

Reply:
You can have it as array.
parameters:
   - name: PROJECT_ID
      type: STRING
      required: true
    - name: Another_parameter
      type: STRING
      required: true